test/cppsuite/CMakeLists.txt

```diff
@@ -55,11 +55,6 @@ create_test_executable(csuite_style_example_test
     CXX
 )
 
-if(ENABLE_TCMALLOC)
```

Comment:
Why we remove this from here? If we are linking at the executable level, should we keep this?

Reply:
create_test_executable is setting the option so we don't need it twice.